nullaway/src/test/java/com/uber/nullaway/NullAwaySerializationTest.java

```diff
@@ -70,8 +102,9 @@ public void cleanup() {
   }
```

Comment:
This `cleanup()` method should be deleted.  JUnit will already take care of deleting the `temporaryFolder` after each test.

Reply:
Oh great, Deleted it.

7587be413f792c10d07af3ff9e0d4c9425ed07fb